paasta_tools/cli/cmds/rollback.py

```diff
@@ -193,36 +252,48 @@ def paasta_rollback(args):
         )
         return 1
 
-    git_shas = get_git_shas_for_service(service, deploy_groups, soa_dir)
+    versions = get_versions_for_service(service, deploy_groups, soa_dir)
     commit = args.commit
+    image_version = args.image_version
+    new_version = DeploymentVersion(sha=commit, image_version=image_version)
     if not commit:
```

Comment:
Do we also want to check that `image_version` is set, given the line in the arg-parser docs:
`However if one is not provided, paasta rollback will instead output a list of valid versions to rollback to.`

Reply:
Ah actually the arg-parser docs already state that it's only _required_ in certain circumstances in the sentence right before it: `If your service has enabled no-commit redeploys, both a commit and the extra metadata is required for paasta rollback to run`, and we have `required=False`.  

I'll add a TODO here for us to actually do the verification *if* we can detect the requested deploy group has no-commit deploys enabled, but think we can leave it w/o for now.